_targets_eval_postprocessing.R

```diff
@@ -1095,12 +1097,19 @@ hub_targets <- list(
   # Write a function that will get hub scores + all the metadata
   # horizon by week, location, forecast_date + eval data alongside it
   # for the models specified in the eval config
+  tar_target(
+    name = boolean_to_pull_locally,
+    command = any(is.null(c(
+      dim(metadata_hub_submissions),
+      dim(metadata_hosp_hub_submissions)
+    )))
```

Comment:
Why does this need to be a separate target?

Reply:
Ah yes, it wasn't originally but it needs to be because of the dependency graph -- the retrospective hub submission files need to be generated locally in order to be passed to `score_hub_submissions()` which looks for a file path with the Hub naming convention (either locally or from github). If you don't set this as a target and pass it to that function, it will not know that this needs to be run first (and the subsequent targets will fail bc they are missing the needed files)

Reply:
Discussed a different sol'n face to face